File: generators/service-retrieveAndRank/templates/python/instrumentation.py
from ibmcloudenv import IBMCloudEnv
from watson_developer_cloud import RetrieveAndRankV1

retrieve_and_rank = RetrieveAndRankV1(
    username=IBMCloudEnv.getString('retrieveAndRank_username'),
    password=IBMCloudEnv.getString('retrieveAndRank_password'))

def getService(app):
    return 'watson-retrieve-and-rank', retrieve_and_rank

File: generators/service-retrieveAndRank/templates/node/instrumentation.js
var IBMCloudEnv = require('ibm-cloud-env');
var RetrieveAndRankV1 = require('watson-developer-cloud/retrieve-and-rank/v1');

module.exports = function(app, serviceManager){
    var retrieveAndRank = new RetrieveAndRankV1({
	    url: IBMCloudEnv.getString('retrieveAndRank_url'),
	    username: IBMCloudEnv.getString('retrieveAndRank_username'),
        password: IBMCloudEnv.getString('retrieveAndRank_password')
    });
    serviceManager.set("watson-retrieve-and-rank", retrieveAndRank);
};